### PR TITLE
Feat: completed list

### PR DESCRIPTION
**Pull request by vinc3nati**

### CODE REVIEW

**Review by lomesshh — APPROVED** [review]
Code seems good 👍

File: components/List/list.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+.list li {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  font-size: inherit;
+  padding: 0.7rem 1.35rem;
+  border-radius: var(--border-radius-3);
+  transition: all 0.3s;
```
Comment (thegeorgenikhil) [inline components/List/list.css:17]: `--border-radius-sm/xs` would be a more meaningful css variable name

File: components/List/list.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+.list li:not(.nested-list):hover {
+  cursor: pointer;
+  background-color: var(--color-tertiary-dark);
+  /* color: var(--color-tertiary); */
```
Comment (thegeorgenikhil) [inline components/List/list.css:23]: commented code

**Review by thegeorgenikhil — APPROVED** [review]

**Review by anuj-kumary — APPROVED** [review]

**Review by PeeyushPrashant — APPROVED** [review]
go ahead

**Pull request merged**